eng/ci/templates/jobs/run-integration-tests-linux.yml

```diff
@@ -21,7 +26,7 @@ jobs:
   - template: /eng/ci/templates/steps/setup-e2e-tests.yml@self
     parameters:
       DotnetVersion: 'net7'
-      UseCoreToolsBuildFromIntegrationTests: $(UseCoreToolsBuildFromIntegrationTests)
+      UseCoreToolsBuild: $(UseCoreToolsBuildFromIntegrationTests)
```

Comment:
Does this work? I am not sure this syntax is available during templating.

Either:
1. Remove and use variable in the `setup-e2e-tests` template
2. Use templating syntax: `${{ variables.UseCoreToolsBuildFromIntegrationTests }}`

Reply:
It does work, reference run: https://dev.azure.com/azfunc/public/_build/results?buildId=167416&view=logs&j=9c11dfa3-b7ef-5307-9644-19345fd1ab28&t=84eb7c97-5ae9-5e1d-3ace-abf7fd7b8b81